internal/proxy/impl.go

```diff
@@ -2588,7 +2589,12 @@ func (node *Proxy) Upsert(ctx context.Context, request *milvuspb.UpsertRequest)
 	insertReceiveSize := proto.Size(it.upsertMsg.InsertMsg)
 	deleteReceiveSize := proto.Size(it.upsertMsg.DeleteMsg)
 
-	rateCol.Add(internalpb.RateType_DMLUpsert.String(), float64(insertReceiveSize+deleteReceiveSize))
+	upsertReceiveSize := insertReceiveSize + it.queryReceiveSize
```

Comment:
what is queryReceiveSize mean?
When we rate limit upsert I think we should evaluate more on delete and insert?